test/utility/misc.c

```diff
@@ -220,20 +192,8 @@ testutil_build_dir(TEST_OPTS *opts, char *buf, int size)
 void
 testutil_make_work_dir(const char *dir)
 {
-    size_t len;
-    char *buf;
-
     testutil_clean_work_dir(dir);
-
-    /* Additional bytes for the mkdir command */
-    len = strlen(dir) + strlen(MKDIR_COMMAND) + 1;
-    if ((buf = malloc(len)) == NULL)
-        testutil_die(ENOMEM, "Failed to allocate memory");
-
-    /* mkdir shares syntax between Windows and Linux */
-    testutil_check(__wt_snprintf(buf, len, "%s%s", MKDIR_COMMAND, dir));
-    testutil_check(system(buf));
-    free(buf);
+    testutil_mkdir(dir);
```

Comment:
Having a wrapper that does nothing except call a different function really means we should get rid of one or the other and just use one function. So maybe get rid of `testutil_make_work_dir` and just have them call `testutil_mkdir`. Same with `testutil_clean_work_dir` above.